README.md

```diff
@@ -106,7 +105,8 @@ With the possibilites GodMode9 provides, not everything may be obvious at first
 
 ### Game file handling
 * __List titles installed on your system__: Press R+A on a /title dir or a subdir below that. This will also work directly for `CTRNAND`, `TWLN` and `A:`/`B:` drives. This will list all titles installed in the selected location. Works best with the below two features.
-* __Build CIAs from NCCH / NCSD (.3DS) / TMD (installed contents)__: Press A on the file you want converted and the option will be shown. Installed contents are found (among others) in `1:/titles/`(SysNAND) and `A:/titles/`(SD installed). Where applicable, you will also be able to generate legit CIAs. Note: this works also from a file search and title listing.
```

Comment:
are you sure you meant to remove "NCCH" from here?

Reply:
The hell? I hate these chromebook's touchpads.

Reply:
I'll fix it when I get home. These school chromebooks are awful.